specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Builds.json

```diff
@@ -388,6 +393,23 @@
         "server"
       ]
     },
+    "BuildEndpoint": {
```

Comment:
Can this be renamed to some generic endpoint, so that you can reuse it as Build Endpoint as well as LogsStreamingEndpoint. Currently, using BuildEndpoint for both doesn't seem right.

Reply:
@visingla-ms renamed to just endpoint here, hopefully thats fine. We use it for upload endpoint and logstream endpoint.